.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@david-salinas this file `.project` does not seem to be needed. could you help remove it from the PR?